SimMuon/GEMDigitizer/plugins/GEMDigiProducer.cc

```diff
@@ -67,9 +105,9 @@ void GEMDigiProducer::produce(edm::Event& e, const edm::EventSetup& eventSetup)
   MixCollection<PSimHit> hits{cf.product()};
 
   // Create empty output
-  auto digis = std::make_unique<GEMDigiCollection>();
-  auto stripDigiSimLinks = std::make_unique<StripDigiSimLinks>();
-  auto gemDigiSimLinks = std::make_unique<GEMDigiSimLinks>();
+  std::unique_ptr<GEMDigiCollection> digis(new GEMDigiCollection());
```

Comment:
please restore the previous pattern for these lines (`auto x = std::make_unique<T>()`)
`new` should be avoided and `make_unique` should be used

Reply:
It has reverted in the previous pattern.